src/components/VFooter/VFooter.vue

```diff
@@ -0,0 +1,103 @@
+<template>
+  <footer class="flex flex-col gap-10 px-6 py-10 lg:px-10 xl:flex-row xl:gap-8">
+    <!-- Logo and links -->
+    <div
+      v-if="isContentMode"
+      class="flex flex-col gap-10 md:flex-row md:items-center md:justify-between xl:flex-grow"
+    >
+      <VLink href="/" class="text-dark-charcoal">
+        <VBrand class="text-[18px]" />
+      </VLink>
+      <nav>
+        <ul class="grid grid-cols-2 gap-6 text-sm sm:flex sm:flex-row sm:gap-8">
+          <li v-for="page in allPages" :key="page.id">
+            <VLink
+              class="text-dark-charcoal"
+              :href="page.link"
+              show-external-icon
+              >{{ $t(`navigation.${page.id}`) }}</VLink
+            >
+          </li>
+        </ul>
+      </nav>
+    </div>
+
+    <!-- Locale chooser and WordPress affiliation graphic -->
+    <div class="flex flex-col justify-between gap-10 sm:flex-row xl:flex-grow">
+      <div
+        class="flex h-10 w-full w-full items-center justify-center rounded-sm border border-dark-charcoal-20 bg-white sm:w-50"
+      >
+        {{ $t('footer.wip') }}
+      </div>
+      <VLink
+        href="https://wordpress.org"
+        class="text-dark-charcoal hover:no-underline"
+      >
+        <i18n
+          tag="div"
+          path="footer.wordpress-affiliation"
+          class="flex h-full flex-row items-center gap-2 text-sm"
+        >
+          <template #wordpress>
+            <WordPress class="aria-hidden" />
```

Comment:
This SVG has a non-`currentColor` fill color, which makes it difficult to change color using CSS. I've replaced it with `currentColor` in the `VHeaderInternal` PR.

Reply:
Fixed in a58435c.